src/app/pages/signing/signing.component.html

```diff
@@ -1,61 +1,73 @@
+        <a class="dropdown-item" (click)="selectFont('Arial')" href="javascript: void(0)">Arial</a>
+        <a class="dropdown-item" (click)="selectFont('Comic Sans MS')" href="javascript: void(0)">Comic Sans MS</a>
+        <a class="dropdown-item" (click)="selectFont('Courier New')" href="javascript: void(0)">Courier New</a>
+        <a class="dropdown-item" (click)="selectFont('Dancing Script')" href="javascript: void(0)">Dancing Script</a>
+        <a class="dropdown-item" (click)="selectFont('Montserrat')" href="javascript: void(0)">Montserrat</a>
+        <a class="dropdown-item" (click)="selectFont('Roboto')" href="javascript: void(0)">Roboto</a>
+        <a class="dropdown-item" (click)="selectFont('Times New Roman')" href="javascript: void(0)">Times New Roman</a>
       </div>
     </div>
-
-    <button type="button" class="btn btn-outline-dark mr-2" id="button-bold">
-      bold
+  
+    <button type="button" (click)="makeBold(inputarea)" class="btn btn-secondary btn-format mr-2" id="button-bold" #b>
+      <b>B</b>
     </button>
-    <button type="button" class="btn btn-outline-dark mr-2" id="button-italics">
-      italics
+    <button type="button" (click)="makeItalics(inputarea)" class="btn btn-secondary btn-format mr-2" id="button-italics">
+      <i>I</i>
     </button>
-
+    <button type="button" (click)="makeUnderline(inputarea)" class="btn btn-secondary btn-format mr-2"
+      id="button-underline">
+      <u>U</u>
+    </button>
+    <button type="button" (click)="makeStrikethrough(inputarea)" class="btn btn-secondary btn-format mr-2"
+      id="button-strikethrough">
+      <s>S</s>
+    </button>
+  
     <div class="btn-group mr-2" role="group" aria-label="Font group">
       <div>
-        <input type="color" id="colorpicker mr-2" name="backgroundPicker" value="#ffffff">
+        <input type="color" class="mr-2" id="colorpicker-font" name="fontColorPicker" value={{textColor}}
+          [(ngModel)]='textColor'>
       </div>
       <div>
-        <input type="color" id="colorpicker mr-2" name="fontColorPicker" value="#000000">
+        <input type="color" class="mr-2" id="colorpicker-background" name="backgroundColorPicker"
+          value={{backgroundColor}} [(ngModel)]='backgroundColor'>
       </div>
     </div>
-    <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#submitConfirmModal">
-      Submit
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#submitConfirmModal">
+      Send
     </button>
   </div>
-  <div class="row">
-    <h3>Preview:</h3>
-  </div>
-  <textarea
-    class="shadow"
-    id="signing-box"
-    autofocus
-  ></textarea>
-  <div class="row">
-    <markdown id="preview-box" data="{{signingBoxText}}"></markdown>
-  </div>
 </div>
 
+
+
+
 <div class="modal fade" id="submitConfirmModal" tabindex="-1" role="dialog" aria-labelledby="submitConfirmModalTitle"
```

Comment:
Improper ID name
**Also change this in Line 74**
```suggestion
<div class="modal fade" id="submit-confirm-modal" tabindex="-1" role="dialog" aria-labelledby="submit-confirm-modal-title"
```